backport/backport.py

```diff
@@ -210,7 +212,7 @@ def main():
     print("")
     print("Done cherry-picking")
     print("Backporting language changes")
-    res = do_command(["python3", ".github/backport-languages.py"])
+    res = do_command(["python3", os.path.dirname(os.path.realpath(__file__)) + "/backport-languages.py"])
```

Comment:
Is there also a way to detect how interpreter was called ? (I had to copy python.exe into python3.exe on my system)

Reply:
Kinda, a bit, bit tricky. And not for this PR :)

One could use `sys.argv[0]`, but that might also be the python file itself. So yeah .. copying is the best option :) Weird that Windows does have a python3.exe btw ...

Reply:
It has a "virtual" python3 exe which directs to windows store. And official python3 installer only provides python.exe.

Reply:
`sys.executable`